spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/reactor/SpanSubscriberTests.java

```diff
@@ -58,7 +59,71 @@ public void setup() {
 				.subscribe(System.out::println);
 
 		then(this.tracer.getCurrentSpan()).isNull();
-		then(spanInOperation.get().getTraceId()).isEqualTo(span.getTraceId());
+		then(spanInOperation.get().getParents().get(0)).isEqualTo(span.getSpanId());
+		then(ExceptionUtils.getLastException()).isNull();
+	}
+
+	@Test public void should_support_reactor_fusion_optimization() {
```

Comment:
javadoc what reactor fusion optimization means and why we are being careful to test it?